pkg/apis/service/extension.go

```diff
@@ -472,3 +477,175 @@ func (h Handler) RouteGetGraph(req RouteGetGraphRequest) (*RouteGetGraphResponse
 		Edges:    edges,
 	}, nil
 }
+
+// CollectionRouteWorkflowStepExecutionRequest is the request for workflow service step execution.
+// It will create or update the service and return the revision config file.
+func (h Handler) CollectionRouteWorkflowStepExecution(req CollectionRouteWorkflowStepExecutionRequest) (any, error) {
```

Comment:
Why service API needs to be aware of workflow execution? I was expecting workflow itself handles deployment using service CRUD APIs.